src/plugins/intel_cpu/tests/unit/transformations/state_concat_sdpa.cpp

```diff
@@ -50,7 +58,31 @@ static std::shared_ptr<ov::Model> makeSDPA(const ov::PartialShape& inputShape, b
         pastv = std::make_shared<ov::op::v8::Gather>(pastv, beam_idx, op::v0::Constant::create(element::i32, {1}, {0}));
         concatK = std::make_shared<ov::op::v0::Concat>(OutputVector{pastk, k}, 2);
         concatV = std::make_shared<ov::op::v0::Concat>(OutputVector{pastv, v}, 2);
-        sdp = std::make_shared<ov::opset13::ScaledDotProductAttention>(q, concatK, concatV, false);
+        if (hasMultiquery) {
+            auto make_multi_query = [&] (const Output<Node>& conat) {
+                auto ShapeOf_219297_autogenerated = makeOP<ov::op::TypeRelaxed<opset1::ShapeOf>>({beam_idx},
+                    {{"type_relax", true}, {"input_data_types", {}}, {"output_data_types", {element::i32}}});
+                auto unsqueeze_Unsqueeze_2 = makeOP<opset1::Unsqueeze>({conat, 2});
+                auto ShapeOf_250564_autogenerated = makeOP<ov::op::TypeRelaxed<opset1::ShapeOf>>(
+                    {conat},
+                    {{"type_relax", true}, {"input_data_types", {}}, {"output_data_types", {element::i32}}});
+                auto Gather_411049 = makeOP<opset8::Gather>({ShapeOf_250564_autogenerated, {2, 3}, 0}, {{"batch_dims", 0}});
+                auto ListConstruct_Concat_2 = makeOP<opset1::Concat>({ShapeOf_219297_autogenerated, {inputShape[1] / 4}, {4}, Gather_411049}, {{"axis", 0}});
+                auto expand_Abs = makeOP<opset1::Abs>({ListConstruct_Concat_2});
+                auto Constant_236480_autogenerated = makeConst(element::u8, ov::Shape({}), 0);
+                auto Broadcast_236481_autogenerated = makeOP<opset1::Broadcast>({{1.000000f},
+                    expand_Abs,
+                    Constant_236480_autogenerated}, {{"mode", "numpy"}});
+                auto expand_Broadcast = makeOP<opset1::Multiply>({unsqueeze_Unsqueeze_2,
+                    Broadcast_236481_autogenerated}, {{"auto_broadcast", "numpy"}});
+                auto ListConstruct_Concat_3 = makeOP<opset1::Concat>({ShapeOf_219297_autogenerated, {inputShape[1]}, Gather_411049}, {{"axis", 0}});
```

Comment:
Could we change the name with `autogenerated` postfix?

Reply:
Done.